### ISSUE TITLE
Failed build does not abort deploy

### ISSUE THREAD

**Issue by Erikvv** [Zenmo/Local4Local#93]
In https://github.com/Zenmo/Local4Local/actions/runs/11834717829/job/32975978150 building the front-end fails yet the deploy continues.

```
 > tsc -b && vite build

src/components/solarfarm/solarfarms-display-edit.tsx(19,57): error TS2345: Argument of type 'number' is not assignable to parameter of type 'AssetType'.
```

**Comment by Erikvv** [comment]
Fixed in a394d1cc8297818fc2e32248d87d84d58367dfdc

**Comment by Erikvv** [comment]
Somehow this change is not in main
